sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -635,7 +635,9 @@ case class DescribeTableCommand(
  * 3. VALUES statement.
  * 4. TABLE statement. Example : TABLE table_name
  * 5. statements of the form 'FROM table SELECT *'
- * 6. Common table expressions (CTEs)
+ * 6. Multi select statements of the following form:
```

Comment:
I think 'nested' is better than 'multi'.

Reply:
`multi` refers to `select * select *`, nested is not the key here.